### ISSUE TITLE
Embed: Accessibility Issues, Placeholder img missing alt

### ISSUE THREAD

**Issue by rnunez80** [Semantic-Org/Semantic-UI-React#4427]
Accessibility Issues
WCAG 2.1 Level A
Alternative (alt) text is read to users utilizing screen readers. Alternative text should be provided to describe what is present in the image so that users who are unable to see have an equal experience as sighted users. Quick tip: If the image is being utilized as a link with no other text, the alternative text should be the location of the link. If the image is of words, the alternative text should be the words contained in the image.

Impact: critical

Violations
WCAG 2.1 Level A: [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
Section 508: 1194.22 (a) Text equivalent for non-text elements
Element Location

Failure Summary
Fix any of the following: Image does not have an alt attribute. To markup the image as decorative, override the semantics by adding role="none" or role="presentation"
